searchcore/src/apps/vespa-feed-bm/document_api_message_bus_bm_feed_handler.cpp

```diff
@@ -0,0 +1,88 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#include "document_api_message_bus_bm_feed_handler.h"
+#include "bm_message_bus.h"
+#include "pending_tracker.h"
+#include <vespa/document/fieldvalue/document.h>
+#include <vespa/document/update/documentupdate.h>
+#include <vespa/documentapi/messagebus/messages/putdocumentmessage.h>
+#include <vespa/documentapi/messagebus/messages/removedocumentmessage.h>
+#include <vespa/documentapi/messagebus/messages/updatedocumentmessage.h>
+#include <vespa/storageapi/messageapi/storagemessage.h>
+
+using document::Document;
+using document::DocumentId;
+using document::DocumentUpdate;
+using storage::api::StorageMessageAddress;
+using storage::lib::NodeType;
+
+namespace feedbm {
+
+DocumentApiMessageBusBmFeedHandler::DocumentApiMessageBusBmFeedHandler(BmMessageBus &message_bus)
+    : IBmFeedHandler(),
+      _name(vespalib::string("DocumentApiMessageBusBmFeedHandler(distributor)")),
+      _storage_address(std::make_unique<StorageMessageAddress>("storage", NodeType::DISTRIBUTOR, 0)),
+      _message_bus(message_bus)
+{
+}
+
+DocumentApiMessageBusBmFeedHandler::~DocumentApiMessageBusBmFeedHandler() = default;
+
+void
+DocumentApiMessageBusBmFeedHandler::send_msg(std::unique_ptr<documentapi::DocumentMessage> msg, PendingTracker& pending_tracker)
+{
+    _message_bus.send_msg(std::move(msg), _storage_address->getRoute(), pending_tracker);
+}
+
+void
+DocumentApiMessageBusBmFeedHandler::put(const document::Bucket& bucket, std::unique_ptr<Document> document, uint64_t timestamp, PendingTracker& tracker)
+{
+    (void) bucket;
+    (void) timestamp;
+    auto msg = std::make_unique<documentapi::PutDocumentMessage>(std::move(document));
+    send_msg(std::move(msg), tracker);
+}
+
+void
+DocumentApiMessageBusBmFeedHandler::update(const document::Bucket& bucket, std::unique_ptr<DocumentUpdate> document_update, uint64_t timestamp, PendingTracker& tracker)
+{
+    (void) bucket;
+    (void) timestamp;
```

Comment:
Is it not better to just drop the naming the variables as they are named in the interface. Keeping them does not add any value.